src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -48,9 +48,9 @@ export function isJupyterNotebook(document: NotebookDocument): boolean;
 export function isJupyterNotebook(viewType: string): boolean;
 export function isJupyterNotebook(option: NotebookDocument | string) {
     if (typeof option === 'string') {
-        return option === JupyterNotebookView;
+        return option === JupyterNotebookView || option === InteractiveWindowView;
     } else {
-        return option.notebookType === JupyterNotebookView;
+        return option.notebookType === JupyterNotebookView || option.notebookType === InteractiveWindowView;
```

Comment:
The changes in this file combined with the package.json interactive/toolbar contribution allow us to reuse literally all our existing variable view code for the interactive window.

Reply:
Great, it's nice when something comes together like that. Sign that we structured something right.